projects/ziti-console-lib/src/lib/ziti-console-lib.module.ts

```diff
@@ -224,7 +224,8 @@ export function playerFactory() {
             useFactory: onAppInit,
             deps: [Injector],
             multi: true
-        }
+        },
+        {provide: APP_BASE_HREF, useValue: '/zac/'}
```

Comment:
I'm not 100% certain about this part.. It feels like the shared lib module (`ziti-console-lib.module`) might not be the correct place to set this.

I think this might be better set in `app.module.ts` in the `app-ziti-console` project, although I'm still not sure if it should be hard coded to `/zac/`, since technically you should be able to extend the shared library and serve it at any `baseHref`.

Reply:
You're right. It shouldn't even be set to a default value in the lib.